context("Check use of simulate() throws and error")
test_that("simulate() returns an error", {
  expect_error(simulate())
})
